wazimap_ng/cache.py

```diff
@@ -1,16 +1,16 @@
-from datetime import datetime
 import logging
+from datetime import datetime
 
 from django.core.cache import cache
 from django.db.models.signals import post_save
 from django.dispatch import receiver
-from django.views.decorators.cache import cache_page, cache_control
-from django.views.decorators.vary import vary_on_headers
 from django.http import Http404
+from django.views.decorators.cache import cache_control
+from django.views.decorators.vary import vary_on_headers
 
-from .profile.models import ProfileIndicator, ProfileHighlight, IndicatorCategory, IndicatorSubcategory, ProfileKeyMetrics, Profile
-from .profile.services import authentication
-from .points.models import Location, Category, Theme
+from wazimap_ng.points import models as points_models
```

Comment:
why did you change the import to a named import?

Reply:
Had a quick discussion with @adieyal on imports, we were thinking named imports would have a better readability vs doing too many imports. Happy to take up a discussion on this front.